.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Node.js & Deno CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
```

Comment:
Please make sure it uses node version from .nvmrc file: https://github.com/Maxim-Mazurok/google-api-typings-generator/blob/d7d95f7306a97e8e13f161ea1ad3aa469683c0e1/.github/workflows/ci-test.yml#L17-L23

Also revert the change to `package-lock.json` please. As per `engines` section in `package.json` this project is expected to run with `"npm": "6.14.8"` which uses old lock format.

And please run `npm run compile` and commit changes from `build` folder as well, they might be needed for some node/js users I guess.